test/hotspot/jtreg/runtime/valhalla/inlinetypes/MonitorEnterTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+* Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+* DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+*
+* This code is free software; you can redistribute it and/or modify it
+* under the terms of the GNU General Public License version 2 only, as
+* published by the Free Software Foundation.
+*
+* This code is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+* version 2 for more details (a copy is included in the LICENSE file that
+* accompanied this code).
+*
+* You should have received a copy of the GNU General Public License version
+* 2 along with this work; if not, write to the Free Software Foundation,
+* Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+*
+* Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+* or visit www.oracle.com if you need additional information or have any
+* questions.
+*/
+
+/**
+* @test TestCloneableValue
+* @library /test/lib
+* @enablePreview
+* @compile TestCloneableValue.java
+* @run main/othervm -XX:LockingMode=0 runtime.valhalla.inlinetypes.TestCloneableValue
```

Comment:
I'm missing the connection between TestCloneableValue and MonitorEnterTest?
Should the \@run line refer to MonitorEnterTest.MyValue?

Reply:
Good catch! This is not the test I used during development, but one that I wrote after, and I clearly did it too quickly.
I've pushed a fix for the test that should address this issue (and a few others in the test itself).